tools/klee/main.cpp

```diff
@@ -220,7 +220,8 @@ class KleeHandler : public InterpreterHandler {
 
   SmallString<128> m_outputDirectory;
 
-  unsigned m_testIndex;  // number of tests written so far
+  unsigned m_numTotalTests;    // number of tests found so far
```

Comment:
Minor nit. These comments probably ought to be updated. Maybe something like this?

```c++
unsigned m_numTotalTests; // Number of test cases received from interpreter
unsigned m_numGeneratedTests; // Number of test cases received and successfully generated
```

I don't have strong opinions here though. I'm happy to merge without the change if you disagree.